11/umbraco-deploy/release-notes.md

```diff
@@ -273,11 +392,76 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
 
-### Deploy 4.1.0 (March 25th 2021)
+### 4.1.0 (March 25th 2021)
 
 *   Management dashboard for triggering and viewing status of deployment operations
 *   Release of Deploy On-Premises
 
-### Deploy 4.0.1 (March 23rd 2021)
+### 4.0.1 (March 23rd 2021)
+
+*   Enabling Deploy 4 to work in new Cloud infrastructure  
+
+</details>
+
+<details>
+
+<summary>Version 2</summary>
+
+### [2.1.6](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.4.2) (January 11th 2022)
+
+*   Improved reliability of extractions triggered from Umbraco Cloud git deployment operations by introducing a new marker file used only on start-up (back-port from 4.4.0).
+
+### [2.1.5](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F2.1.5) (December 7th 2021)
+
+*   Fixed issue relating to deployment of Data Types with prevalues on Umbraco 7. [#20](https://github.com/umbraco/Umbraco.Cloud.Issues/issues/20) [#45](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/45) [#89](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/89)
+
+### 2.1.4
+*   Resolution of issue with failed extractions on vNext infrastructure.
+
+### 2.0.18 + 2.1.3 (July 6th 2021)
+
+*   Updating Cloud vNext related configuration
+
+</details>
+
+
+## [Deploy Contrib](https://github.com/umbraco/Umbraco.Deploy.Contrib)
+
+{% hint style="info" %}
+Deploy Contrib contains community contributions for Umbraco Deploy targetted for version 8 and above.
+
+It offers connectors for the most popular Umbraco community packages. These are used by Deploy to aid with the deployment and transferring of content/property-data between environments.
+{% endhint %}
+
+### Deploy Contrib 11.0.1 (February 14th 2023)
+
+*   Block grid editor support and fix for null reference exception (10+)
+*   Further caching for deploy operations.
+
+### Deploy Contrib 10.1.1 (February 14th 2023)
+
+*   Block grid editor support and fix for null reference exception (10+)
+*   Further caching for deploy operations.
+
+### Deploy Contrib 9.1.1 (February 14th 2023)
+
+*   Block grid editor support and fix for null reference exception (10+)
+*   Further caching for deploy operations.
+
+### Deploy Contrib 4.2.0, 9.1.0 and 10.1.0 (September 7th 2022)
+
+*   Used caching in value connectors to improve performance.
+*   Fixed issue with transfer of nested content string values using item templates (i.e. commencing with "{{") that were erroneously considered as JSON.
+
+### Deploy Contrib 4.1.2, 9.0.2, 10.0.1 (July 12th 2022)
+
+*   Optimized existence checks in connectors
+
+
+### Deploy Contrib 4.1.1 (April 26th 2022)
+
+*   Fixes issue with deployment of nested content properties within doc type grid editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```

Comment:
```suggestion
*   Fixes issue with deployment of nested content properties within Document Type Grid Editor values. [#82](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/82)
```

Reply:
Doc type grid editor is the name of a package: https://our.umbraco.com/packages/backoffice-extensions/doc-type-grid-editor/ Which is the reason why I did not change the name 🤔 

Should it still be changed or?

Reply:
Ooh! Didn't realize 😄 Then ignore my suggestion!